revolv/static/adjustment.scss

```diff
@@ -0,0 +1,25 @@
+@import "utils";
+@import "revolv-settings";
+
+.save-button {
+	@include revolv-hover-button($revolv-dark-blue, $revolv-dark-blue);
+	width: 97%;
```

Comment:
Oh, I think it would be better to do width: 100% and box-sizing: border-box in order to make it more precise